dapr/proto/runtime/v1/dapr.proto

```diff
@@ -82,6 +82,9 @@ service Dapr {
   // Gets the state for a specific actor.
   rpc GetActorState(GetActorStateRequest) returns (GetActorStateResponse) {}
 
+  // Gets the state for a specific actor.
+  rpc DeleteActorState(DeleteActorStateRequest) returns (google.protobuf.Empty) {}
```

Comment:
A bit of a nit, but for good hygiene consider creating a response type `DeleteActorStateResponse` which is empty, instead of `google.protobuf.Empty`. In the future there may be the need to add properties, and creating a custom type already makes this easier (incl. on SDKs)